app/controllers/QuestionnaireController.scala

```diff
@@ -162,4 +166,8 @@ class QuestionnaireController(applicationClient: ApplicationClient) extends Base
     }
   }
 
+  private def isCivilServant(fastPassDetails: Option[FastPassDetails]) = {
```

Comment:
isCivilServant(fastPassDetails: Option[FastPassDetails]) logic is duplicated in SchemePreferencesController as well, shall we centralise this to some where, perhaps FastPassDetails object